instana/collector/helpers/runtime.py

```diff
@@ -181,7 +181,7 @@ def _collect_runtime_snapshot(self, plugin_data):
 
             if 'AUTOWRAPT_BOOTSTRAP' in os.environ:
                 snapshot_payload['m'] = 'Autowrapt'
-            elif 'INSTANA_MAGIC' in os.environ:
+            elif '/tmp/.instana/python' in sys.path:
```

Comment:
There are better practices than having hardcoded paths or file names, IMHO. It would be better to have this path as a global configuration variable, and then we can set it once to be used across the code.

In addition, I also found the `INSTANA_MAGIC` variable in the `instana/instrumentation/fastapi_inst.py` file, so I guess that can also be removed.

Reply:
Valid point, the hardcoding should be eliminated.